README.md

```diff
@@ -11,7 +11,7 @@ This repository contains the consensus specifications related to the Ethereum ex
 | Version and Code Name | Block No. | Released | Incl EIPs | Specs | Blog |
 |-----------------------|-----------|----------|-----------|-------|-------|
 | Shanghai | TBD | TBD | TBD| [Specification](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/shanghai.md) | |
-| Paris | TBD | TBD | TBD| [Specification](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md) | |
+| Paris | 15537394 | 2022-09-15 | [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675) <br/> [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399) | [Specification](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md) | [Blog](https://blog.ethereum.org/2022/08/24/mainnet-merge-announcement) |
```

Comment:
Hmm - I wonder if we should add the block here or use the TTD instead? It might be confusing to have the block number if we then add the `mergeNetSplit` block as we did on Sepolia?

Reply:
Once the merge happens and is finalized, a node really shouldn't use pow TTD or block number to *sync* to the head given that alternative histories at that TTD or block number could be crafted.

assuming you have direction from the beacon chain, you can premise the logic transition upon TTD or EL block number.

I don't see much issue putting block number here, as long as it is deeply known that to safely sync you need a beacon chain.

Reply:
I guess the easy fix is to just put both here? Something like this, even if it's really long?
```suggestion
| Paris | 15537394 (TTD: 58750000000000000000000) | 2022-09-15 | [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675) <br/> [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399) | [Specification](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md) | [Blog](https://blog.ethereum.org/2022/08/24/mainnet-merge-announcement) |
```

Reply:
> I don't see much issue putting block number here, as long as it is deeply known that to safely sync you need a beacon chain.

I don't think this table here ever was used for syncing, given it doesn't record block hashes or anything of bigger certainty.